### PR TITLE
Add support for running using `tmt`

### PR DESCRIPTION
**Pull request by jamacku**

### CODE REVIEW

**Comment by jamacku** [comment]
@lzaoral, could you please review this PR? Thanks

File: .packit.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@ jobs:
     trigger: pull_request
     targets:
       - fedora-all
+
+  - job: tests
+    trigger: pull_request
+    targets:
+      - fedora-all
+    skip_build: true
```
Comment (lzaoral) [inline .packit.yml:17]: This line is not necessary.  The test will automatically inherit the builds done in by the `copr_build` jobs.  The package will not be built twice.
```suggestion
```

File: tests/alternatives/main.fmf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+summary: Alternatives integration tests
+component: alternatives
+framework: beakerlib
+contact: Jan Macku <jamacku@redhat.com>
+tier: 1
+require:
+  - gcc
+  - make
+  - beakerlib
```
Comment (lzaoral) [inline tests/alternatives/main.fmf:9]: nit: It is not necessary to list `beakerlib` here, it was already specified by the `framework`.

File: tests/alternatives/main.fmf [hunk 3]
```diff
@@ -0,0 +1,14 @@
+summary: Alternatives integration tests
+component: alternatives
+framework: beakerlib
+contact: Jan Macku <jamacku@redhat.com>
+tier: 1
+require:
+  - gcc
```
Comment (lzaoral) [inline tests/alternatives/main.fmf:7]: nit: you may want to sort this list

**Review by lzaoral — CHANGES_REQUESTED** [review]
The test logs contain following errors:
```
:: [ 12:14:36 ] :: [   PASS   ] :: NEW_FOLLOWER	link: /tmp/tmp.nmaS2fKnwG/link_a/follower_a (Expected 0, got 0)
basename: missing operand
Try 'basename --help' for more information.

:: [ 12:14:36 ] :: [   PASS   ] :: Mode: (Assert: 'auto' should equal 'auto')
...
./test-alternatives.sh: line 310: rlPhase: command not found
```

Unfortunately, `beakerlib` is not compatible with `set -e` so we can't catch such errors this way. :/

File: .packit.yml (hunk 1 above)
Comment (lzaoral) [reply .packit.yml:17]: At also means, that the current TF jobs use the latest package from Fedora and not the build from Packit.

File: tests/alternatives/main.fmf (hunk 2 above)
Comment (jamacku) [reply tests/alternatives/main.fmf:9]: Thank you, I missed this.

File: tests/alternatives/main.fmf (hunk 3 above)
Comment (jamacku) [reply tests/alternatives/main.fmf:7]: I actually like the long list a bit more in this case.

File: tests/alternatives/test-alternatives.sh [hunk 4]
```diff
@@ -7,7 +7,8 @@
 TEST="Test Alternatives"
 
 # Package being tested
-PACKAGE="chkconfig"
+PACKAGE="alternatives"
+TEST_BIN="${TEST_PATH:-}${PACKAGE}"
```
Comment (lnykryn) [inline tests/alternatives/test-alternatives.sh:11]: is the ':-' neccesary?

File: .packit.yml (hunk 1 above)
Comment (jamacku) [reply .packit.yml]: Good catch, thanks

File: tests/alternatives/test-alternatives.sh (hunk 4 above)
Comment (jamacku) [reply tests/alternatives/test-alternatives.sh]: fixed

**Comment by jamacku** [comment]
> The test logs contain following errors:
> 
> ```
> ./test-alternatives.sh: line 310: rlPhase: command not found
> ```

@lzaoral Thank you for noticing this error! It should be fixed now.

**Review by lzaoral — CHANGES_REQUESTED** [review]
Thank you for the update, @jamacku!  Unfortuantely, the `basename` error in the last phase is still present:
```
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
::   Dynamic Follower Remove
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:: [ 13:12:53 ] :: [  BEGIN   ] :: NEW	link: link_a	Prio: 10	Family:  :: actually running 'alternatives --altdir /tmp/tmp.G6lNXfm3rY --admindir /tmp/tmp.Cj5OGoKIB9 --install /tmp/tmp.Q0AgP9k3Z8/main_link TEST /tmp/tmp.Q0AgP9k3Z8/link_a/main 10 --slave /tmp/tmp.Q0AgP9k3Z8/follower_link STEST /tmp/tmp.Q0AgP9k3Z8/link_a/follower '
:: [ 13:12:53 ] :: [   PASS   ] :: NEW	link: link_a	Prio: 10	Family:  (Expected 0, got 0)
:: [ 13:12:54 ] :: [  BEGIN   ] :: NEW_FOLLOWER	link: /tmp/tmp.Q0AgP9k3Z8/link_a/follower_a :: actually running 'alternatives --altdir /tmp/tmp.G6lNXfm3rY --admindir /tmp/tmp.Cj5OGoKIB9 --add-slave TEST /tmp/tmp.Q0AgP9k3Z8/link_a/main /tmp/tmp.Q0AgP9k3Z8/follower_link STEST /tmp/tmp.Q0AgP9k3Z8/link_a/follower_a'
:: [ 13:12:54 ] :: [   PASS   ] :: NEW_FOLLOWER	link: /tmp/tmp.Q0AgP9k3Z8/link_a/follower_a (Expected 0, got 0)
:: [ 13:12:54 ] :: [  BEGIN   ] :: NEW_FOLLOWER	link: /tmp/tmp.Q0AgP9k3Z8/link_a/follower_a :: actually running 'alternatives --altdir /tmp/tmp.G6lNXfm3rY --admindir /tmp/tmp.Cj5OGoKIB9 --remove-slave TEST /tmp/tmp.Q0AgP9k3Z8/link_a/main STEST'
:: [ 13:12:54 ] :: [   PASS   ] :: NEW_FOLLOWER	link: /tmp/tmp.Q0AgP9k3Z8/link_a/follower_a (Expected 0, got 0)
basename: missing operand
Try 'basename --help' for more information.

:: [ 13:12:54 ] :: [   PASS   ] :: Mode: (Assert: 'auto' should equal 'auto')
:: [ 13:12:54 ] :: [   PASS   ] :: Highest Priority: (Assert: 'link_a' should equal 'link_a')
:: [ 13:12:54 ] :: [   PASS   ] :: Selected: (Assert: 'link_a' should equal 'link_a')
:: [ 13:12:54 ] :: [   PASS   ] :: Follower: (Assert: '' should equal '')
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
::   Duration: 1s
::   Assertions: 7 good, 0 bad
::   RESULT: PASS (Dynamic Follower Remove)
```

**Pull request merged**